qiskit/circuit/library/__init__.py

```diff
@@ -206,6 +206,7 @@
    UCRXGate
    UCRYGate
    UCRZGate
+   QftGate
```

Comment:
Side comment: I think this should be `QFT` since "Qft" is not a word (compare e.g. RVGate vs. PauliGate)

Reply:
Done!